docs/sources/flow/_index.md

```diff
@@ -33,6 +33,17 @@ Components allow for reusability, composability, and focus on a single task.
 * Use expressions to bind components together to build a programmable pipeline.
 * Includes a UI for debugging the state of a pipeline.
 
+{{< param "PRODUCT_NAME" >}} is a [distribution][] of the OpenTelemetry
+Collector by Grafana Labs.
+Each distribution offers a different collection of components and capabilities.
+{{< param "PRODUCT_NAME" >}} is a distribution that includes dozens of
```

Comment:
```suggestion
{{< param "PRODUCT_NAME" >}} includes dozens of
```

Is it necessary to define what the distributions do?

Reply:
Not sure, we were on the fence but it might be worth to explain what makes a/our distribution different.